doc/userguide/rules/ldap-keywords.rst

```diff
@@ -314,4 +314,38 @@ This is an example of a signature that would alert if a ``success`` result code
 
 .. container:: example-rule
 
-  alert tcp any any -> any any (msg:"Test LDAP success at last index"; :example-rule-emphasis:`ldap.responses.result_code:success,-1;` sid:1;)
\ No newline at end of file
+  alert tcp any any -> any any (msg:"Test LDAP success at last index"; :example-rule-emphasis:`ldap.responses.result_code:success,-1;` sid:1;)
+
+ldap.responses.error_message
```

Comment:
Why did we pick `error_message` ? Why not `response.message` as in the json output ? Or `result_message` to match more `result_code`

Reply:
I just used https://hepia.infolibre.ch/reseauxI/ldap.asn1.pdf as reference for the name.

Which of these two options you presented do you think is better?

Reply:
Let's go for `ldap.responses.message` which matches more the current json schema, and if you want you can add an alias `ldap.responses.error_message`

Reply:
And if you do add the alias, ensure to document that, too :)